app/serializers/v1/customer_serializer.rb

```diff
@@ -57,6 +57,9 @@ def billing_configuration
       elsif model.payment_provider&.to_sym == :gocardless
         configuration[:provider_customer_id] = model.gocardless_customer&.provider_customer_id
         configuration.merge!(model.gocardless_customer&.settings || {})
+      elsif model.payment_provider&.to_sym == :adyen
```

Comment:
Style/CaseLikeIf: Convert `if-elsif` to `case-when`. (https://rubystyle.guide#case-vs-if-else)